src/test/SwapRouterNoChecks.sol

```diff
@@ -43,5 +43,7 @@ contract SwapRouterNoChecks is PoolTestBase {
             data.key.currency1.settle(manager, data.sender, uint256(int256(-delta.amount1())), false);
             data.key.currency0.take(manager, data.sender, uint256(int256(delta.amount0())), false);
         }
+
+        return "";
```

Comment:
what happens to gas if instead of this you just name the return param?
like `(bytes memory returnData)` and then leave it unassigned?

Reply:
just intrigued haha

Reply:
+37 gas, I just wanted to get rid of the compiler warning